frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -158,6 +158,7 @@ def send(self, smtp_server_instance: SMTPServer = None):
 			return
 
 		with SendMailContext(self, smtp_server_instance) as ctx:
+			ctx.fetch_smtp_server()
```

Comment:
This is now required. AFAIK `SendMailContext` is internal so not treating this as breaking. 

https://sourcegraph.com/search?q=context%3Aglobal+SendMailContext&patternType=standard&sm=0&groupBy=repo&filters=%5B%5B%22lang%22%2C%22Python%22%2C%22lang%3Apython%22%5D%5D